web/themes/custom/badcamp2017/sass/components/_featured.scss

```diff
@@ -1,16 +1,18 @@
-.section-featured {
-  background-color: transparent;
-  .section--inner {
-    background-image: url("../images/homepage-bg-image.jpg");
+
+.path-frontpage {
+  #zone-highlighted {
+    background-image: url('../images/homepage-bg-image.jpg');
```

Comment:
Is zone highlighted spanning the entire width of the page with no padding? It should be according to comp.